Zend/zend_execute.c

```diff
@@ -1656,6 +1656,75 @@ static inline zend_brk_cont_element* zend_brk_cont(int nest_levels, int array_of
 	return jmp_to;
 }
 
+static inline void zend_do_namespaced_special_function_call(zend_execute_data *execute_data, zend_function *fbc, zval *value, zend_uchar op_type) {
+	zend_bool has_arg = value && !Z_ISUNDEF_P(value);
+
+	EX(call) = zend_vm_stack_push_call_frame(ZEND_CALL_NESTED_FUNCTION, fbc, has_arg, NULL, NULL, EX(call));
+
+	if (has_arg) {
+		zval *arg = ZEND_CALL_VAR(EX(call), EX(opline)->result.var);
```

Comment:
This looks fishy ... for SEND opcodes `result.var` contains the call stack slot for the argument, but for special functions `result.var` should be the return value, right?